src/Entity/ModuleVersion.php

```diff
@@ -21,6 +22,25 @@ class ModuleVersion extends AbstractBaseEntity
     #[ORM\ManyToMany(targetEntity: Installation::class, inversedBy: 'moduleVersions')]
     private Collection $installations;
 
+    public function __toString(): string
+    {
+        return $this->getVersion();
+    }
+
+    /**
+     * @throws Exception
+     */
+    public function display(int $style): string
```

Comment:
Using integers doesn't relay what the parameter does. Using a value like `detail` does that better. Also please document the parameter with a proper phpDoc block https://docs.phpdoc.org/3.0/guide/getting-started/what-is-a-docblock.html